packages/app-store/index.ts

```diff
@@ -0,0 +1,27 @@
+import * as example from "./_example";
+import * as applecalendar from "./applecalendar";
+import * as dailyvideo from "./dailyvideo";
+import * as googlecalendar from "./googlecalendar";
+import * as googlevideo from "./googlevideo";
+import * as huddle01video from "./huddle01video";
+import * as jitsivideo from "./jitsivideo";
+import * as office365calendar from "./office365calendar";
+import * as office365video from "./office365video";
+import * as tandemvideo from "./tandemvideo";
+import * as zoomvideo from "./zoomvideo";
+
+const appStore = {
+  example,
+  applecalendar,
+  dailyvideo,
+  googlecalendar,
+  googlevideo,
+  huddle01video,
+  jitsivideo,
+  office365calendar,
+  office365video,
+  tandemvideo,
+  zoomvideo,
+};
+
+export default appStore;
```

Comment:
This is the most important file for adding app to the App Store. This is basically the entry point of it all. We can just comment out any app to make it disappear from the web app.

Reply:
This would be the place to refactor to make it dependant on the database to have the possibility to be managed by admins, enabling or disabling for ee.